media-tv/tvbrowser-bin/tvbrowser-bin-4.0.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop
+
+DESCRIPTION="Themeable and easy to use TV Guide - written in Java"
+HOMEPAGE="https://www.tvbrowser.org"
+MY_PN="tvbrowser"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://sourceforge/project/tvbrowser/TV-Browser%20Releases%20%28Java%208%20and%20higher%29/${PV}/${MY_PN}_${PV}_bin.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="virtual/jdk:1.8"
+
+S="${WORKDIR}"/"${MY_P}"
+
+src_install() {
+	# Copy files and directories
+	dodir /opt/"${P}"
+	insinto /opt/"${P}"
+	doins -r *
+
+	# Generate launcher
+	exeinto /usr/bin
```

Comment:
Executables for stuff from /opt belong in /opt/bin.

Also the typical way of doing that would be:

    into /opt
    dobin ...

Reply:
Yes, but in this case the ebuild is generating the launcher, thus using exeinto and not dobin. However, I've adapted the path to /opt/bin now.